### ISSUE TITLE
Adding particle effects/smoke like the original?

### ISSUE THREAD

**Issue by Hexaae** [ptitSeb/stuntcarremake#12]
Only one thing is missing in this great remake: smoke when off-road, and particles when you move on the edge of the tracks...
Will these be implemented for a faithful remake? :)

**Comment by ptitSeb** [comment]
Yes those 2 effects are missing. I want to add them, but not sure when.

Note that also the Championship mode is missing, only practice is implemented for now.

**Comment by Hexaae** [comment]
Nice. Looking forward for a complete remake then... ;)
